packages/flame-css/README.md

```diff
@@ -0,0 +1,36 @@
+# Flame css
+
+`flame-css` is an alternative offering to `flame`.
+
+It offers a sass codebase and an easy to consume css file.
```

Comment:
I would avoid mentioning Sass in the intro. Perhaps later down as an "Unsupported" feature that we might drastically change or remove at some point.